src/app/settings/organization-plans.component.html

```diff
@@ -92,7 +92,7 @@ <h2 class="mt-5">{{'chooseYourPlan' | i18n}}</h2>
                 </small>
             </ng-template>
             <span *ngIf="selectableProduct.product != productTypes.Free">
-                <ng-container *ngIf="selectableProduct.basePrice">
+                <ng-container *ngIf="selectableProduct.basePrice && !acceptingSponsorship">
```

Comment:
I don't think this should have `acceptingSponsorship`, right?

Reply:
This one **does** need it. Otherwise it will show " $3.33 /month, includes 6 users" which wouldn't be true.